R/use_beachmat.R

```diff
@@ -0,0 +1,37 @@
+#' Use beachmat
+#'
+#' Creates \code{src/} and adds needed packages to \code{DESCRIPTION}.
+#'
+#' For more details see the 'Linking' vignette, available by running
+#' \code{vignette("linking", package = "beachmat")}.
+#' @importFrom usethis use_directory use_git_ignore
+#' @export
+use_beachmat <- function() {
```

Comment:
Four space indenting is the norm in this package.

Reply:
Fixed